build.go

```diff
@@ -364,6 +364,34 @@ func Build(builder Builder, options ...Option) {
 			return
 		}
 	}
+	logBindings(ctx.Platform.Bindings, logger)
+}
+
+func logBindings(bindings Bindings, logger poet.Logger) {
+	numBindings := len(bindings)
+	if numBindings > 0 {
+		if numBindings == 1 {
+			logger.Info("\n1 Binding Identified:")
```

Comment:
Should this be debug level logging? It seems like it could be annoying if you add a binding intended to pass data to a single buildpack and every buildpack in the build lists every binding.

Also it seems like we typically log at the debug level in libcnb unless something goes wrong, allowing the buildpacks themselves to design the default output